tedana/selection/select_comps.py

```diff
@@ -61,10 +61,59 @@ def selcomps(seldict, mmix, mask, ref_img, manacc, n_echos, t2s, s0, olevel=2,
         Indices of rejected (non-BOLD) components in `mmix`
     midk : list
         Indices of mid-K (questionable) components in `mmix`
+        These components are typically removed from the data during denoising
     ign : list
         Indices of ignored components in `mmix`
+        Ignored components are considered to have too low variance to matter.
+        They are not processed through the accept vs reject decision tree and
+        are NOT removed during the denoising process
     """
 
+    """
+    There are sections of this code that calculate metrics that are used in
```

Comment:
What do you think of moving these metrics into the sphinx documentation, rather than as code comments ?

Reply:
I think that's the eventual goal, but it's a bit tricky when the code is only half-commented. Until there's full documentation of how each metric is or isn't used, making something seem definitive int he sphinx documentation could cause more harm than help. For example, there are some metrics which I thought would be used one way, but they were transformed into another metric & used in more component selection steps than I realized. In that case, saying Metric X is used in selection step Y would be wrong. My hope is, once the full process is better commented in the code, it will be relatively easy to summarize those comments in end-user documentation.